techbuzz/src/main/java/com/sivalabs/techbuzz/users/web/controllers/RegistrationController.java

```diff
@@ -82,4 +86,48 @@ private void sendVerificationEmail(HttpServletRequest request, UserDTO userDTO)
         Map<String, Object> paramsMap = Map.of("", userDTO.name(), "verificationUrl", verificationUrl);
         emailService.sendEmail("email/verify-email", paramsMap, to, subject);
     }
+
+    @GetMapping("/resentVerification")
```

Comment:
The "resentVerification" related controller handler methods can be created inside a separate `ResentVerificationController` class.

Reply:
done